### PR TITLE
v0.7.2 Release Candidate

### PR DESCRIPTION
**Pull request by safl**
The branch should have been named ``v0.7.1-rc1``, thus nothing that this is for ``v0.7.2-rc1`` despite the name of the branch.

### CODE REVIEW

File: docs/contributing/release-checklist.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+  - See the ``git log`` for previous commits bumping the version-number, to
+    check that you bump everywhere needed.
+
+* Update man-pages and Bash completion-scripts
+
+  - Build and install the version-bumped **xNVMe**
+  - Then run: ``make gen-man-pages gen-bash-completions``
+  - Commit the man-pages with message: ``docs(man): update for vX.Y.Z``
+  - Commit the Bash-completions-scripts message: ``feat(toolbox/completions): update for vX.Y.Z``
```
Comment (karlowich) [inline docs/contributing/release-checklist.rst:26]:
```suggestion
  - Commit the Bash-completions-scripts with message: ``feat(toolbox/completions): update for vX.Y.Z``
```

File: docs/contributing/release-checklist.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+* Update ``CONTRIBUTORS.rst``
+
+  - Get the list of contributors with: ``git log main..HEAD~1 --pretty=format:"%an <%aE>" | sort | uniq``
+  - Update the lists accordingly
+  - Commit changes with the message: ``CONTRIBUTORS: update for vX.Y.Z``
+
+* Update ``CHANGELOG.rst``
+
+  - TODO: Describe the content here...
```
Comment (karlowich) [inline docs/contributing/release-checklist.rst:36]: This todo should probably be removed

File: docs/contributing/release-checklist.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  - Add the artifacts generated on the **tag** (src-archive,
+    test-verify-results, Python sdist)
+
+* Publish the Python package
+
+  - TODO: describe this
+
+* Publish the Rust crate
+
+  - TODO: describe this
```
Comment (karlowich) [inline docs/contributing/release-checklist.rst:69]: These todos should be filled out or added as github issues

**Review by karlowich — CHANGES_REQUESTED** [review]
Three small comments

File: docs/contributing/release-checklist.rst (hunk 2 above)
Comment (safl) [reply docs/contributing/release-checklist.rst:36]: Replaced with a brief description

File: docs/contributing/release-checklist.rst (hunk 3 above)
Comment (safl) [reply docs/contributing/release-checklist.rst:69]: Right, added here: https://github.com/OpenMPDK/xNVMe/issues/382

Comment (safl) [reply docs/contributing/release-checklist.rst]: And removed the todos :)

**Review by karlowich — APPROVED** [review]
Great!

**Pull request merged**